src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -334,3 +334,17 @@
             {}
             tokens)
     (update-vals #(prettify-balance currency-symbol %))))
+
+(defn resolve-receiver-networks
+  [{:keys [prefix testnet-enabled? goerli-enabled?]}]
```

Comment:
Can we please avoid putting utility methods in these generically named files with a bunch of random logic? 🙏 

It's a much better practice to categorise the utilities to some degree and keep certain functionality in each file. 

See:
https://github.com/status-im/status-mobile/pull/19579/files#r1558338145

In this particular case I suggest doing the same (no need to move the others), just add the file
src/status_im/contexts/wallet/common/utils/networks.cljs and move this functionality there instead 🙏 

Unless there is a better approach? albeit I certainly don't think we should continue with this  current approach :)